cli/main.rs

```diff
@@ -758,12 +762,19 @@ async fn test_command(
     let mut pretty_coverage_reporter = PrettyCoverageReporter::new();
     for script_coverage in filtered_coverage {
       let module_specifier = ModuleSpecifier::resolve_url_or_path(&script_coverage.url)?;
+      println!("{}", module_specifier.to_string());
       let source_file = global_state
         .file_fetcher
         .fetch_cached_source_file(&module_specifier, Permissions::allow_all());
 
+      // FIXME this always fails
```

Comment:
This should return the source map when given a compiled typescript source, shouldn't it?

Reply:
Getting the compiled source file works just fine here.

Reply:
So we inline source maps now which is a slight problem for future me that will need the original source locations for lcov reports but for now I'll just count the covered lines of the compiled source when available as that's really all we need in a summary report.